src/librustc_mir/dataflow/impls/borrows.rs

```diff
@@ -90,6 +97,19 @@ impl<'a, 'tcx> Borrows<'a, 'tcx> {
                     borrows.insert(idx);
                 }
             }
+
+            fn visit_statement(&mut self,
+                               block: mir::BasicBlock,
+                               statement: &mir::Statement<'tcx>,
+                               location: Location) {
+                if let mir::StatementKind::EndRegion(region_scope) = statement.kind {
+                    error!("statement {:?} {:?}", statement, statement.source_info.span);
+                    if let Some(span) = self.region_span_map.insert(ReScope(region_scope), statement.source_info.span) {
```

Comment:
I think the tidy pass is complaining about this line being too long. (We have a hard 100-character limit on line length.)